app/views/layouts/_top_bar.html.erb

```diff
@@ -49,7 +49,7 @@
         </a>
 
         <div class="crayons-header__menu" id="crayons-header__menu">
-          <button type="button" class="crayons-header__menu__trigger" id="member-menu-button" aria-label="Navigation menu">
+          <button type="button" class="crayons-header__menu__trigger" id="member-menu-button" aria-label="Navigation menu" aria-expanded="false">
```

Comment:
`aria-expanded` needs to start off as `"false"`. From there, interactions from JS take over.